sormas-backend/src/main/java/de/symeda/sormas/backend/common/StartupShutdownService.java

```diff
@@ -355,6 +355,15 @@ private void createDefaultUsers() {
 			userService.persist(admin);
 			userUpdateEvent.fire(new UserUpdateEvent(admin));
 
+
+			if (!configFacade.getCreateDefaultUsers()) {
```

Comment:
Move this new code and the admin creation code to above the calls to retrieve regions, districts, communities etc. as that's unnecessary if no default users are created anyway